src/lib/postTask.ts

```diff
@@ -15,9 +16,10 @@ import {
 import { IUIKitViewSubmitIncomingInteraction } from "@rocket.chat/apps-engine/definition/uikit/UIKitIncomingInteractionTypes";
 import { ICreateTaskState } from "../facade/IClickUpService";
 import { IUser } from "@rocket.chat/apps-engine/definition/users";
-import { getAccessTokenForUser } from "../storage/users";
+import { getAccessTokenForUser, get_clickup_uid } from "../storage/users";
 import { ModalsEnum } from "../enums/Modals";
 import { HttpStatusCode } from '@rocket.chat/apps-engine/definition/accessors';
+import { sendDirectMessage } from "./message";
 
 export async function postTask({
```

Comment:
I think it will be nice if you make this function smaller, maybe split some code in to small functions

Reply:
Done in next PR.